src/input.rs

```diff
@@ -206,6 +207,11 @@ impl EditorCamInputMessage {
                         .get_entity(*pointer)
                         .and_then(|entity| pointer_interactions.get(entity).ok())
                         .and_then(|interaction| interaction.get_nearest_hit())
+                        // Since `bevy` 0.17.3:
+                        //
+                        // If the current hit is on a window, we cannot use the `hit.position` as an anchor
+                        // as the `hit.position` is in viewport coordinates.
```

Comment:
```suggestion
                        // as the `hit.position` is in screen coordinates.
```

I have a doubt on which is the correct nomenclature here.